Description: Don't run failing tests.
 That way we can still benefit by running all the other tests.
Author: Jeremy Bicha <jbicha@ubuntu.com>
Date: Fri, 05 May 2017 08:26:36 -0500
Bug: https://bugzilla.gnome.org/782215
     https://bugzilla.gnome.org/753836
---
 clutter/tests/conform/Makefile.am |    2 --
 1 file changed, 2 deletions(-)

--- a/clutter/tests/conform/Makefile.am
+++ b/clutter/tests/conform/Makefile.am
@@ -28,8 +28,6 @@ actor_tests = \
 	actor-offscreen-limit-max-size \
 	actor-offscreen-redirect \
 	actor-paint-opacity \
-	actor-pick \
-	actor-shader-effect \
 	actor-size \
 	$(NULL)
 
